### PR TITLE
Update README with new certificate creation process

### PR DESCRIPTION
**Pull request by davidmccoy**
Apple has updated its developer portal so I updated the README to include the proper steps for creating and downloading a .cer file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,15 +279,16 @@ The key code for this purpose is:
...
+Login to the [iOS Provisioning Portal (App IDs)](https://developer.apple.com/account/ios/identifier/bundle).
 
-Configure the appropriate certificate for push notifications and download the
-certificate:
+Select your app bundle, scroll down and click "Edit." This will bring you to a page that will allow you to setup and configure services for your app. Scroll down until you see the service labeled “Push Notifications.” Depending on whether you’re creating a Development or Production certificate, click the appropriate “Create Certificate” button.
 
-![Downloading the Push Notification
-Certificate](images/Downloading_the_Push_Notification_Certificate.png)
+![Creating the Push Notification
+Certificate](https://cdn-images-1.medium.com/max/1600/1*zHi-WR44OGYtOdFCy79jZw.png)
```
Comment (stevenharman) [inline README.md]: Can we add this image to this repo? Or capture one that we can? Hot-linked images tend to break as time goes on.

**Comment by davidmccoy** [comment]
Image has been added. 😄

**Comment by stevenharman** [comment]
Merged. Thanks, @davidmccoy! 👍

**Comment by davidmccoy** [comment]
No problem. I spent a good part of my day figuring the new process out and though I’d share.

**Pull request merged**